core/runtime/TRTEngine.cpp

```diff
@@ -201,9 +201,10 @@ TRTEngine::TRTEngine(
     }
     num_io = std::make_pair(inputs_size, outputs);
   }
-
 #ifndef NDEBUG
-  this->enable_profiling();
+  if (util::logging::get_logger().get_reportable_log_level() == util::logging::LogLevel::kDEBUG) {
```

Comment:
Could just remove this if you feel its not necessary

Reply:
Ok, I removed